src/commands/config/list.js

```diff
@@ -1,16 +1,31 @@
 // @flow
 import * as options from '../../utils/options';
 import { BoltError } from '../../utils/errors';
+import * as yarn from '../../utils/yarn';
+import { DEPENDENCY_TYPE_FLAGS_MAP } from '../../constants';
 
-export type ConfigListOptions = {};
+export type ConfigListOptions = {
+  cwd?: string
+};
 
 export function toConfigListOptions(
   args: options.Args,
   flags: options.Flags
 ): ConfigListOptions {
-  return {};
+  if (
+    args.length !== 0 ||
+    Object.keys(DEPENDENCY_TYPE_FLAGS_MAP).length !== 0
+  ) {
+    throw new BoltError('Invalid subcommand, try bolt config list');
+  }
+  return { cwd: options.string(flags.cwd, 'cwd') };
 }
 
 export async function configList(opts: ConfigListOptions) {
-  throw new BoltError('Unimplemented command "config list"');
+  let cwd = opts.cwd || process.cwd();
+  try {
+    await yarn.cliCommand(cwd, 'config list');
```

Comment:
this can be `await yarn.cliCommand(cwd, 'config', ['list']);`

as yarn.cliCommand is :

```js
export async function cliCommand(
  cwd: string,
  command: string = '',
  spawnArgs: Array<string> = []
) {
  const localYarn = path.join(await getLocalBinPath(), 'yarn');

  return await processes.spawn(localYarn, [command, ...spawnArgs], {
    cwd,
    tty: true
  });
}
```

Reply:
makes sense. I will change it.